src/components/nested_screens/profile/TradeHistoryScreen.js

```diff
@@ -1,11 +1,19 @@
 import React from "react";
 import { View, Text, StyleSheet } from "react-native";
 import { THEME } from "../../../constants/Theme";
+import HeaderContainer from "../../reusable_components/HeaderContainer";
+import CustomTable from "../../reusable_components/CustomTable";
 
 export default function TradeHistoryScreen() {
   return (
     <View style={styles.container}>
-      <Text style={styles.text}>Welcome to the trade history screen!</Text>
+      <HeaderContainer
+        headerText="Trade History"
+        bodyText="A detailed view of all transactions made by your job."
```

Comment:
"A detailed view of all transaction made by your job(s). "

Reply:
Addressed with latest commit